secure/key-manager/src/tests.rs

```diff
@@ -75,12 +72,10 @@ fn setup_secure_storage(
 
 impl Node {
     fn setup(config: &NodeConfig) -> Self {
-        let (storage, db_reader_writer) = storage_service::init_libra_db(config);
-        let storage_service =
-            storage_service::start_storage_service_with_db(&config, storage.clone());
-        maybe_bootstrap_db::<LibraVM>(db_reader_writer, config)
-            .expect("Db-bootstrapper should not fail.");
-        let executor = Executor::new(SyncStorageClient::new(&config.storage.address).into());
+        let (storage, db_handle) = DbReaderWriter::wrap(LibraDB::new(&config.storage.dir()));
```

Comment:
I think it'd be nice if we don't introduce the "DB handle" concept before we have a type `DbHandle`.